SQA_AT/src/test/java/api/StoreTest.java

```diff
@@ -0,0 +1,52 @@
+package api;
+
+import client.StoreClient;
+import io.restassured.response.Response;
+import model.Order;
+import org.testng.annotations.Test;
+
+import java.net.HttpURLConnection;
+
+import static org.testng.Assert.assertEquals;
+
+public class StoreTest {
+    @Test
+    public void getStoreInventory() {
+        StoreClient storeClient = new StoreClient();
+        Response response = storeClient.getStoreInventory();
+        assertEquals(response.statusCode(),  HttpURLConnection.HTTP_OK, "Error - incorrect status code");
```

Comment:
Would be nice to implement Fluent Assertions design pattern here
https://refactoring.pl/en/how-to-refactor-tests-towards-fluent-assertion-pattern/

Also take a look on assertj library and soft assertions using assertj lib
https://www.baeldung.com/introduction-to-assertj
https://www.javacodegeeks.com/2015/09/assertjs-softassertions-do-we-need-them.html